src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -96,5 +104,18 @@ private bool RemoteNode_MessageReceived(NeoSystem system, Message message)
             }
             return true;
         }
+
+        private static void OnCommitting(NeoSystem system, Block block, DataCache snapshot, IReadOnlyList<Blockchain.ApplicationExecuted> applicationExecutedList)
+        {
+            var serializedData = JsonSerializer.SerializeToUtf8Bytes(applicationExecutedList);
+            var hash = ComputeSha256Hash(serializedData);
```

Comment:
Here, you are proposing the `Sha256` (of all Executed List) on every payload for PrepareRequest.